### DISCUSSION TITLE
Turborepo: Use react email in ./packages

### DISCUSSION THREAD

**Discussion by fxmb** [resend/react-email#682] [Q&A]
Amazing project!

I would like to use react E-Mail in turborepo and import it into my backend which is in apps/backend. 

However, I don't want to bloat my server and insert react email directly there. I would like to import it as a package but also be able to start the local dev server for emails in the packages folder. 

I seem to have problems installing the dependencies correctly in packages using yarn workspace packages/emails and then run yarn workspace packages/email dev. It doesn't install the lock files correctly so the server does not start. Is there an example repo that successfully implemented this logic? Thanks 🙏

**Comment by fxmb** [comment]
Can anyone help here? It's a huge blocker atm. Are there any examples of this implemented?

**Comment by Tobjoern** [comment]
Yes help would be highly appreciated.
We started using react-email, because we thought it would be an easy to integrate, but that's not at all the case.
Right now we aren't even able to build the project because of it.
